<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Differ</class>
 <widget class="QWidget" name="Differ">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>55</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Стоимость кредита:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Max ежемесячный платеж, руб.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Min ежемесячный платеж, руб.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Переплата по кредиту, руб.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Общая выплата по кредиту, руб.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEdit_platMax">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {

border: 2px solid gray;
}

background-color: white;</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_platMin">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {

border: 2px solid gray;
}

background-color: white;</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_overPlat">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {

border: 2px solid gray;
}

background-color: white;</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_result">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {

border: 2px solid gray;
}

background-color: white;</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QTableWidget" name="tableWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>741</width>
         <height>220</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>182</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>182</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Сумма платежа, руб.</string>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="background">
         <color>
          <red>119</red>
          <green>255</green>
          <blue>216</blue>
         </color>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Основной долг, руб.</string>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="background">
         <color>
          <red>119</red>
          <green>255</green>
          <blue>216</blue>
         </color>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Проценты, руб.</string>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="background">
         <color>
          <red>119</red>
          <green>255</green>
          <blue>216</blue>
         </color>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Остаток долга, руб.</string>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="background">
         <color>
          <red>119</red>
          <green>255</green>
          <blue>216</blue>
         </color>
        </property>
       </column>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_OK">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgb(119, 255, 216);
color: gray;
border: 2px solid gray;
}
QPushButton:pressed {
    background-color: rgb(180, 61, 255);
}</string>
       </property>
       <property name="text">
        <string>ОК</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_OK</sender>
   <signal>clicked()</signal>
   <receiver>Differ</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
